subprojects/plugins/src/integTest/groovy/org/gradle/api/tasks/JavaExecDebugIntegrationTest.groovy

```diff
@@ -55,7 +55,7 @@ class JavaExecDebugIntegrationTest extends AbstractIntegrationSpec {
 
         expect:
         def failure = executer.withTasks(taskName).withStackTraceChecksDisabled().runWithFailure()
-        failure.assertHasErrorOutput('ERROR: transport error 202: connect failed:')
+        failure.assertHasErrorOutput('ERROR: transport error 202: ')
```

Comment:
I found this accidentally while running the test suite. The `connect failed` part of the message is actually JDK dependent. GraalVM returns a different string.